modules/shared/components/molecules/PostFooterCreation/PostFooterCreation.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { IPostFooterCreation } from './IPostFooterCreation';
+import styles from './PostFooterCreation.module.css';
+import Privacy from '../Privacy/Privacy';
+import Divider from '../../atoms/Divider/Divider';
+
+/** 
+* @Notice you will use the handleChange functions to get and use the value of the toggler and the select
+* @ThingsNeededToBeDoneInParentComponent
+const [togglerIsChecked,setTogglerIsChecked] = useState(false);
+const handleTogglerChange = (e: React.ChangeEvent<HTMLInputElement>):void => { setTogglerIsChecked(e.target.checked) }<Test>;
+const handleSelectChange = (e: React.ChangeEvent<HTMLSelectElement>):void => {console.log(e.target.value)};
+*/
+const PostFooterCreation: FC<IPostFooterCreation.IProps> = ({
+  handleSelectChange,
+  togglerIsChecked,
+  handleTogglerChange,
+  disabled = true,
+  handleSubmitClick,
+  handleCancelClick,
```

Comment:
`togglerIsChecked ` 👍  very clear, anyone who reads the name without digging into the code will understand it is a boolean. 
`handleSelectChange `: The name is very generic, what we select ( Date, time, icon)
`handleTogglerChange `:  What does it toggle, we need to include what  the toggleer change in the name

Reply:
I did the rename thing